server/gossip/gossip.go

```diff
@@ -27,6 +28,7 @@ var (
 	join       = flag.Slice("gossip.join", []string{}, "The nodes to join/gossip with. Ex. '1.2.3.4:1991,2.3.4.5:1991...'")
 	nodeName   = flag.String("gossip.node_name", "", "The gossip node's name. If empty will default to host_id.'")
 	secretKey  = flag.String("gossip.secret_key", "", "The value should be either 16, 24, or 32 bytes.")
+	logLevel   = flag.String("gossip.log_level", "warn", "The desired log level. Logs with a level >= this level will be emitted. One of {'fatal', 'error', 'warn', 'info', 'debug'}")
```

Comment:
could we maybe default this to "" and ignore it if it's unset? Otherwise we have to flags to set the logging level in the app which could be confusing

Reply:
Done. I also clarify it that this flag is for gossip package only in the flag comments.